site/js/geopatterns-demo.js

```diff
@@ -116,3 +117,8 @@ const GeopatternsApp = {
 const app = Vue.createApp(GeopatternsApp)
 
 app.mount('#app');
+
+$.getJSON(TEXT_URL, function(data) {
```

Comment:
`$` is jQuery, I am afraid jQuery is not available here and this will not work. Vue.js is available and its means should be used. Vue.js tutorial is rather well written.